### PR TITLE
Add Dutch (nl)

### PR DESCRIPTION
**Pull request by onderay**
# Explain your changes

_Suppose there is a related issue with enough detail for a reviewer to understand your changes fully. In that case, you can omit an explanation and instead include either “Fixes #XX” or “Updates #XX” where “XX” is the issue number._

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

File: nl [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                "email_input_label": "E-mailadres",
+                "email_input_error_invalid": "Voer een geldig e-mailadres in,",
+                "email_input_error_required": "E-mailadres is verplicht,",
+                "email_input_error_duplicate_email_found": "Dit e-mailadres is al ingeschreven,",
+                "continue_button": "Inschrijven",
+                "success_heading": "Succes! Welkom bij de berichtgeving",
+                "credit_powered_by": "Mogelijk gemaakt door"
+            },
+            "verification_code_sms": {
+                "description": "${generated_code} is uw eenmalige code om u aan te melden bij ${business_name}\n\n @${business_name} #${generated_code]"
```
Comment (DaveOrDead) [inline nl:281]: @onderay I wonder if the spreadsheet needs updating, possibly I missed it, but we seem to be getting a mix of business name and `receiver_domain` - this one should be `@${receiver_domain} #${generated_code]`

Comment (onderay) [reply nl]: @DaveOrDead updated

File: nl [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+                "email_input_label": "E-mailadres",
+                "email_input_error_invalid": "Voer een geldig e-mailadres in,",
+                "email_input_error_required": "E-mailadres is verplicht,",
+                "email_input_error_duplicate_email_found": "Dit e-mailadres is al ingeschreven,",
+                "continue_button": "Inschrijven",
+                "success_heading": "Succes! Welkom bij de berichtgeving",
+                "credit_powered_by": "Mogelijk gemaakt door"
+            },
+            "verification_code_sms": {
+                "description": "${generated_code} is uw eenmalige code om u aan te melden bij ${receiver_domain}\n\n @${receiver_domain} #${generated_code]"
```
Comment (DaveOrDead) [inline nl:281]: @onderay sorry should be 

```${generated_code} is uw eenmalige code om u aan te melden bij ${business_name}\n\n@${receiver_domain} #${generated_code}```

previous was missing `$business_name` and had a `]` at the end

**Review by DaveOrDead — CHANGES_REQUESTED** [review]

File: nl (hunk 2 above)
Comment (onderay) [reply nl]: @DaveOrDead updated

**Review by DaveOrDead — APPROVED** [review]

**Pull request merged**